### PR TITLE
Cr ac delete item

### LINKED ISSUE #11: 11. Como usuario, quiero poder eliminar artículos de mi lista de compras, pues  no quiero que mi lista esté llena de artículos que no quiero comprar en el futuro.
Es posible que nuestros usuarios cometan un error al escribir un elemento que desean comprar. O pueden decidir que no necesitan comprar un elemento de nuevo en el futuro. Vamos a ayudarlos a mantener la lista limpia!

AC:

* El usuario puede eliminar un elemento de la lista de compra
* Antes de eliminar el elemento, preguntale al usuario si esta seguro que quiere borrar el elemento para prevenir accidentes al borrar.
* Al eliminar el elemento, la entrada en la base de datos asociada a ese elemento debe ser eliminada.

### PR DESCRIPTION
**Pull request by alxmcr**
## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|  ✓ | :sparkles: New feature     |
|  ✓ | :hammer: Refactoring       |

## Description

As a user, I want to be able to delete items from my shopping list so that my list isn’t cluttered with items I don’t want to buy in the future.

## Related Issue

Closes #11 

## Acceptance Criteria

- [x]  User can remove an item from the shopping list
- [x]  Before deleting the element, ask the user if he is sure he wants to delete the element to prevent accidents when deleting.
- [x]  When deleting the element, the entry in the database associated with that element must be eliminated.

### Before

image

### After

image

## Testing Steps / QA Criteria

- Join a shopping list with a token.
- Go to "List"
- Click "Delete" then the app asks you "Are you sure you want to delete this item <item-name>" then you'll have two options: "OK" and "Cancel". If you click "OK", the item will be deleted. On another hand, if you click "Cancel", the question will be closed.

### CODE REVIEW

**Review by macosta08 — APPROVED** [review]
Excellent

File: src/components/Prompt/index.jsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+function Prompt({ isShowed, children, deleteAction, toggleModal }) {
+  return (
+    isShowed && (
+      <div className="prompt" aria-modal="true" role="dialog" id="dialog1">
+        <br />
```
Comment (Lukaina) [inline src/components/Prompt/index.jsx]: Great that they have implemented accessibility. 👏

**Review by Lukaina — APPROVED** [review]
Excellent work, @CrisRuedaP and @alxmcr, I love that you have implemented accessibility and the way you solved the issue, it's great. Congratulations! Approved!

File: src/components/Prompt/index.jsx (hunk 1 above)
Comment (alxmcr) [reply src/components/Prompt/index.jsx]: Thank you so much, Verónica! 😅

**Comment by indiragp** [comment]
Deben usar algo parecido a esto: https://www.w3schools.com/jsref/met_win_alert.asp para confirmar que van a eliminar el elemento. 
Lo ideal es que solo se pueda eliminar un elemento a la vez, y que el usuario solo pueda ver la seccion del elemento que se va a eliminar. En la implementacion actual se le puede dar eliminar a todos los elementos al mismo tiempo.

File: src/components/Prompt/index.jsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+function Prompt({ isShowed, children, deleteAction, toggleModal }) {
+  return (
+    isShowed && (
+      <div className="prompt" aria-modal="true" role="dialog" id="dialog1">
```
Comment (indiragp) [inline src/components/Prompt/index.jsx]: Id's must be unique. If you open multiple "delete" buttons. All the "prompts" id's will be the same "dialog1"

**Review by indiragp — APPROVED** [review]
Great job! Thanks for the prompt update!

File: src/components/Prompt/index.jsx (hunk 2 above)
Comment (alxmcr) [reply src/components/Prompt/index.jsx]: Thanks for your comments!
Well, I changed our `Prompt` component to `windows.confirm` 😄 .

**Pull request merged**